### PR TITLE
Update project initiailization steps in quick-start.md

### PR DESCRIPTION
**Pull request by SriramBharadwaj1**
## Description of Problem

Ambiguity in creating a new vue application.
In line number 25 of vuejs/docs/src/guide/quick-start.md, the directory in which the npm init command is to be run is not mentioned. 
Front-end beginners might run the command in some directory and get confused. 

## Proposed Solution
To resolve this ambiguity, I propose we mention that the command be run in the project root folder as the current working directory.

## Additional Information
little things like this make the documentation more beginner-friendly and encourage them

### CODE REVIEW

File: src/guide/quick-start.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ footer: false
 
 In this section we will introduce how to scaffold a Vue [Single Page Application](/guide/extras/ways-of-using-vue#single-page-application-spa) on your local machine. The created project will be using a build setup based on [Vite](https://vitejs.dev) and allow us to use Vue [Single-File Components](/guide/scaling-up/sfc) (SFCs).
 
-Make sure you have an up-to-date version of [Node.js](https://nodejs.org/) installed, then run the following command in your command line (without the `>` sign):
+Make sure you have an up-to-date version of [Node.js](https://nodejs.org/) installed, then run the following command in your command line with the project root folder as the current working directory (without the `>` sign):
```
Comment (yyx990803) [inline src/guide/quick-start.md:25]: This isn't accurate, since there is no project root folder before you run the tool. The root folder is created by the tool.

Maybe reword it to "in a directory where you intend to create the project"

**Review by yyx990803 — CHANGES_REQUESTED** [review]

File: src/guide/quick-start.md (hunk 1 above)
Comment (SriramBharadwaj1) [reply src/guide/quick-start.md:25]: sure sir, thank you for correcting me. I shall revise the changes

**Comment by SriramBharadwaj1** [comment]
some inaccuracy is found in the current pr I made. 
a revised pull request will be made.
